playbooks/scenario44_rds_instance.yaml

```diff
@@ -4,37 +4,12 @@
   hosts: localhost
   vars:
     prefix: rds-
-    volume_type: "ultrahigh"
+    test_router_name: "Apimon-test-rds-router"
```

Comment:
keep small letters if there's no specific reason for big ones

Reply:
fixed